dataflow/Cargo.toml

```diff
@@ -5,9 +5,10 @@ authors = ["The Distributary Developers"]
 
 [dependencies]
 buf_redux = "0.6.1"
-evmap = "2.0.2"
+evmap = { git = "https://github.com/ms705/rust-evmap" }
```

Comment:
I assume this is to get `evmap` with `rahashmap` so we can evict random keys?

Reply:
Yes, though we can perhaps merge those changes into upstream `evmap` now that rahashmap has been updated. I'll make a PR.